.github/styles/config/vocabularies/Polkadot/accept.txt

```diff
@@ -24,15 +32,20 @@ Karura
 Khala
 Kusama
 [Ll]iveness
+libp2p
 [Mm]ultilocation
+NFT
+NPoS
 [Oo]mninode
 [Pp]arachain
 [Pp]arachain's
 [Pp]arachains
+ParaID
 parathread
 Paseo
 Phala
 Polkadot
+Polkadot OpenGov
```

Comment:
I think adding just OpenGov is enough

Reply:
Marking this as resolved as we'll be using Polkadot OpenGov moving forward